app/Grouptopics/Repositories/Account/AccountEloquent.php

```diff
@@ -1,19 +1,22 @@
 <?php
-namespace Grouptopics\Libraries;
+namespace Grouptopics\Repositories\Account;
 
-use \Input as Input;
-use \Auth as Auth;
-use \User as User;
-use \Validator as Validator;
-use \Hash as Hash;
-use \Redirect as Redirect;
+use Grouptopics\Repositories\EloquentRepository;
 
-class Account
+
+class AccountEloquent extends EloquentRepository implements AccountInterface
```

Comment:
But its in the extend. It generally makes more sense to suffix with what
the object type is

A. Scherer

On Thu, Jan 1, 2015 at 10:43 PM, Eric Van Johnson <notifications@github.com>
wrote:

> In app/Grouptopics/Repositories/Account/AccountEloquent.php
> <https://github.com/sdphp/grouptopics.org/pull/85#discussion-diff-22405388>
> :
>
> >
> > -class Account
> > +
> > +class AccountEloquent extends EloquentRepository implements AccountInterface
>
> I use Eloquent as the naming convention so that you know its using the
> Eloquent Libraries. I like to identify that in the name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sdphp/grouptopics.org/pull/85/files#r22405388>.
>